kura/org.eclipse.kura.broker.artemis.xml/OSGI-INF/metatype/org.eclipse.kura.broker.artemis.xml.BrokerInstance.xml

```diff
@@ -47,7 +47,7 @@
 			description="User configuration in the format: user=password|role1,role2,...|TextArea"
 			type="String"
 			cardinality="1"
-			required="true" />
+			required="false" />
```

Comment:
Why the required is set now to false?

Reply:
Since the ComponentUtil.getObjectClassDefinition has been fixed.The OCD validation can be excited correctly.  If required="true"  the validation would be failed at first running.